cmd/harbor/root/user/elevate.go

```diff
@@ -28,14 +29,22 @@ func ElevateUserCmd() *cobra.Command {
 				userId = utils.GetUserIdFromUser()
 			}
 
-			// Todo : Ask for the confirmation before elevating the user to admin role
+			var confirm bool
 
-			err = runElevateUser(userId)
-
-			if err != nil {
-				log.Errorf("failed to elevate user: %v", err)
+			huh.NewConfirm().
```

Comment:
```go
func confirmElevation() (bool, error) {
	var confirm bool
	|
	|
	|
	.
	.
	.

	return confirm, nil
}
```


Create a separate confirmation model in the 'views' directory and name the file 'confirmation.go'.

Reply:
Yes. I created confirmation.go file.
Please review it @amands98